main/fossil/APKBUILD

```diff
@@ -1,16 +1,17 @@
 # Contributor: Paul Onyschuk <blink@bojary.koba.pl>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=fossil
-pkgver=2.5
-pkgrel=1
+pkgver=2.6
+pkgrel=0
 pkgdesc="Simple, high-reliability, distributed software configuration management"
-url="http://www.fossil-scm.org"
+url="https://www.fossil-scm.org"
 arch="all"
-license="BSD"
+license="BSD-2-Clause"
+options="!check" # missing dependencies; broken tests
```

Comment:
which dependencies missing? maybe add `checkdepends` with required ones?

Reply:
I don't recall now, but I seem to remember they weren't all in aports anyway.